ee/codegen/src/__test__/nodes/code-execution-node.test.ts

```diff
@@ -102,4 +103,43 @@ describe("CodeExecutionNode", () => {
       );
     });
   });
+  describe("with runtime set", () => {
+    it.each<"PYTHON_3_11_6" | "TYPESCRIPT_5_3_3">([
+      "PYTHON_3_11_6",
+      "TYPESCRIPT_5_3_3",
+    ])("should generate the correct standalone file %s", async (override) => {
+      workflowContext = workflowContextFactory({
+        codeExecutionNodeCodeRepresentationOverride: "STANDALONE",
+      });
+
+      const overrideInputValue: NodeInputValuePointerRule = {
+        type: "CONSTANT_VALUE",
+        data: {
+          type: "STRING",
+          value:
+            override == "TYPESCRIPT_5_3_3"
+              ? "async function main(inputs: {\n\n}): Promise<number> {\n\n  return;\n}"
+              : "def main(inputs):\n  return",
```

Comment:
[nit] I always lean towards a second test case arg over using conditionals in tests